### PR TITLE
Fix TL1_tensorflow-dali_test

### PR DESCRIPTION
**Pull request by JanuszL**
- sets a version of TF that works with the current version of ResNet50 TensorFlow example
- adjusts TF version tested for CUDA 10

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It fixes TL1_tensorflow-dali_test

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     sets a version of TF that works with the current version of ResNet50 TensorFlow example
     adjusts TF version tested for CUDA 10
 - Affected modules and functionalities:
     TL1_tensorflow-dali_test
     setup_packages.py
 - Key points relevant for the review:
     NA
 - Validation and testing:
     current test applies
 - Documentation (including examples):
     NA

**JIRA TASK**: *[NA]*

### CODE REVIEW

File: qa/TL1_tensorflow-dali_test/test.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ do_once() {
     CUDA_VERSION=$(echo $(nvcc --version) | sed 's/.*\(release \)\([0-9]\+\)\.\([0-9]\+\).*/\2\3/')
 
     # install TF 2.x
```
Comment (jantonguirao) [inline qa/TL1_tensorflow-dali_test/test.sh:15]:
```suggestion
    # install TF 2.2.*
```

**Review by jantonguirao — APPROVED** [review]

File: qa/TL1_tensorflow-dali_test/test.sh (hunk 1 above)
Comment (JanuszL) [reply qa/TL1_tensorflow-dali_test/test.sh]: Done

Comment (awolant) [reply qa/TL1_tensorflow-dali_test/test.sh]: Isn't it 2.3.1 now? Or am I missing something?

Comment (JanuszL) [reply qa/TL1_tensorflow-dali_test/test.sh]: You are right. Fixed again.

**Review by awolant — APPROVED** [review]

**Pull request merged**